test/test_core.py

```diff
@@ -5900,6 +5900,16 @@ def test_fs_rename_on_existing(self, args):
       self.set_setting('FORCE_FILESYSTEM')
     self.do_runf('fs/test_fs_rename_on_existing.c', 'success', emcc_args=args)
 
+  @parameterized({
+    '': ([],),
+    'nodefs': (['-DNODEFS', '-lnodefs.js'],),
+    'noderawfs': (['-sNODERAWFS'],)
+  })
```

Comment:
Since this seem to be a common pattern I wonder if we should add `also_with_both_nodefs`?